### PR TITLE
21. As a user, I want the "Add Items" page to look visually appealing

### LINKED ISSUE #45: 21. As a user, I want the "Add Items" page to look visually appealing
AC:

- style the "Add Items" page according to the mockup

### PR DESCRIPTION
**Pull request by Maeesha-Rahman**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Updated the layout / styles for AddItem page according to the mockup. I created custom radio buttons with css to resemble the ones in the mockup. 

## Related Issue

closes #45 

## Acceptance Criteria

style the "Add Items" page according to the mockup

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

Screen Shot 2021-02-25 at 7 39 30 PM

### After

Screen Shot 2021-02-25 at 7 38 58 PM

## Testing Steps / QA Criteria

1. pull from branch `mr-add-items-page-ui`
2. npm start
3. navigate to Add Item page
4. check that the UI resembles mockup but is using the colours/fonts in the notion documentation (what we have in `variables.scss` )

### CODE REVIEW

File: src/components/AddItem/AddItem.js [hunk 1]
```diff
@@ -74,41 +79,44 @@ const AddItem = () => {
           onChange={onGroceryItemInputChange}
           required
```
Comment (thekidnamedkd) [inline src/components/AddItem/AddItem.js:90]: Did we want to put a min/max for characters on input? I think we discuss 3 min/12 max as kind of a simple "error" handler for breakpoints.

File: src/components/AddItem/AddItem.js [hunk 2]
```diff
@@ -74,41 +79,44 @@ const AddItem = () => {
...
+              type="radio"
+              name="daysToPurchase"
+              onChange={onRadioInputChange}
+              checked={daysToPurchase === 30}
+            />
+            <label htmlFor="not-soon">Not Soon</label>
+          </div>
         </fieldset>
-        <input type="submit" value="Add Item" />
+        <input type="submit" value="Add to Shopping List" />
```
Comment (thekidnamedkd) [inline src/components/AddItem/AddItem.js:125]: This is better!

File: src/components/AddItem/AddItem.js [hunk 3]
```diff
@@ -74,41 +79,44 @@ const AddItem = () => {
...
       </form>
+      <div className="image-container">
+        <WineDog />
```
Comment (thekidnamedkd) [inline src/components/AddItem/AddItem.js:128]: Wine Dogs 4 Life!

**Review by thekidnamedkd — APPROVED** [review]
Min/max suggestion is an easy change if we want to implement! Great job Maeesha!

**Review by wlcreate — APPROVED** [review]
Great job! Looks good to me 😁

**Pull request merged**